src/lib/examples/getSchemaUiJson.ts

```
@@ -53,6 +53,12 @@
     return [resolveOneOfProperty(property, useExample)]
   }
 
+  if (property.meta?.hasPrefixItems && property.properties && Array.isArray(property.properties)) {
+    return property.properties.map(prop => {
```

ESLint

[error] Expected parentheses around arrow function argument having a body with curly braces.

(style/arrow-parens)